Resources/Prototypes/Entities/Objects/Misc/subdermal_implants.yml

```diff
@@ -213,3 +218,49 @@
         - SubdermalImplant
         - HideContextMenu
         - MacroBomb
+
+
+- type: entity
+  parent: BaseSubdermalImplant
+  id: CritRattleImplant
+  name: critrattle implant
+  description: sends a message with your location to the syndicate radio channel when you fall into a critical state.
+  noSpawn: true
+  components:
+    - type: SubdermalImplant
+      permanent: true
+    - type: TriggerOnMobstateChange
+      mobState: Critical
+    - type: Rattle
+      message: "deathrattle-implant-critical-message"
+
+
+- type: entity
+  parent: BaseSubdermalImplant
+  id: DeathRattleImplant
+  name: deathrattle implant
+  description: triggers a fake bomb countdown and sends a message with your location to the syndicate radio channel when you die.
+  noSpawn: true
+  components:
+    - type: SubdermalImplant
+      permanent: true
+    - type: TriggerOnMobstateChange
+      mobState: Dead
+    - type: OnUseTimerTrigger
+      delay: 10
+      initialBeepDelay: 0
+      beepSound: /Audio/Machines/Nuke/general_beep.ogg
+      params:
+        volume: -2
+    - type: Rattle
+      message: "deathrattle-implant-dead-message"
```

Comment:
I would change this so that the DeathRattleComp handles mobstate triggers so you only need a singular implant.

Reply:
I made TriggerOnMobstateChange accept a list of MobStates to allow multiple triggers in one implant